### DISCUSSION TITLE
UWP Template - any alternatives for BitmapDecoder/BitmapEncoder on WASM

### DISCUSSION THREAD

**Discussion by darenm** [unoplatform/uno#11303] [Q&A] [answered]
Hi there - I am porting an app from WPF that directly manipulates bitmaps - rotating the raw image data and saving as new files. I see that `BitmapDecoder` and `BitmapEncoder` have not been ported to WASM - does anyone have any recommendations for alternatives?

**Comment by jeromelaban** [comment, marked as answer]
SkiaSharp is generally a good solution to this kind of image manipulation.

**Reply by darenm** [reply]
Thanks - that's the approach I settled on. I'll add some examples here once I have something working.
